enterprise/api/v1/views.py

```diff
@@ -200,6 +210,93 @@ def course_enrollments(self, request, pk):
 
         return Response(serializer.errors, status=HTTP_400_BAD_REQUEST)
 
+    @detail_route(methods=['post'], permission_classes=[permissions.IsAuthenticated])
+    @permission_required('enterprise.can_enroll_learners', fn=lambda request, pk: pk)
+    def enterprise_learners(self, request, pk):
```

Comment:
I think we'll want to call this `enroll_enterprise_learners`? Since I suspect we'll eventually want to unenroll them, etc?

Reply:
Generally I want to avoid having verbs in endpoints, that's the point of the HTTP verbs. I don't see that as part of the ACs anywhere, or as part of the miro board spec, it's also not possible right now with the manage_leaners form, so not sure that's something we'll eventually want to even do, but if we decide to I think a DELETE call would be appropriate rather than changing the route name.